oam/oamcpp/oamcache.h

```diff
@@ -42,9 +43,14 @@ class OamCache
     mtime = 0;
   }
 
-  dbRootPMMap_t getDBRootToPMMap();
-  dbRootPMMap_t getDBRootToConnectionMap();
-  PMDbrootsMap_t getPMToDbrootsMap();
+  int getClosestPM(int dbroot); // who can access dbroot's records for read requests - either owner or us.
+  int getClosestConnection(int dbroot); // connection index to owner's PM or ours PM - who can access dbRoot.
+  int getOwnerConnection(int dbroot); // connection index to owner's PM.
+  int getOwnerPM(int dbroot); // Owner's PM index.
+  std::vector<int> getPMDBRoots(int PM); // what DBRoots are owned by given PM.
+  std::vector<int> getAllDBRoots(); // get all DBRoots.
+  bool isAccessibleBy(int dbRoot, int pmId);
+  bool isOffline(int dbRoot); // not registered in map.
```

Comment:
Is it make sense to make this methods `const`?

Reply:
Would it require changes more than marking method(s) const?